cluster/examples/kubernetes/ceph/create-external-cluster-resources.sh

```diff
@@ -9,6 +9,7 @@ set -Eeuo pipefail
 #############
 
 : "${CLIENT_CHECKER_NAME:=client.healthchecker}"
+: "${RGW_POOL_PREFIX:=default}"
```

Comment:
We need to document the values for this. If it's RHCS this is the likely default. If it's a rook cluster, it's likely `my-store`.

Reply:
It's just a place holder for https://github.com/rook/rook/pull/5388 to know what to do. We will take care of that in the other PR.